src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, verbose: bool = True, **source_data):
         self.recording_extractor = self.Extractor(**source_data)
         self.subset_channels = None
         self.verbose = verbose
+        self.es_key = None  # For automatic metadata extraction
```

Comment:
This is usually something that has to be manually overridden at the call to `run_conversion` to allow custom metadata. For automatically fetched electrical series metadata, I think it would help to have this automatically set and propagate without user input